hearth/sig/lib/hearth/telemetry/otel.rbs

```diff
@@ -4,6 +4,18 @@ module Hearth
       def initialize: () -> void
     end
 
+    class OTelTracerProvider < TracerProviderBase
+      def initialize: () -> void
+    end
+
+    class OTelTracer < TracerBase
+      def initialize: (untyped tracer) -> void
```

Comment:
I think you can add types to these. Tracer and Span

Reply:
Not needed per offline convo